src/pybind/mgr/dashboard/frontend/src/app/shared/components/copy2clipboard-button/copy2clipboard-button.component.ts

```diff
@@ -1,31 +1,33 @@
-import { Directive, ElementRef, HostListener, Input, OnInit, Renderer2 } from '@angular/core';
+import { Component, HostListener, Input } from '@angular/core';
 
 import { detect } from 'detect-browser';
 import { ToastrService } from 'ngx-toastr';
 
-@Directive({
-  selector: '[cdCopy2ClipboardButton]'
+@Component({
+  selector: 'cdCopy2ClipboardButton',
```

Comment:
All current selectors follow ```kebab-case``` (hyphen-case), such as: cd-my-component-selector.
For consistency this should be sth like: cd-copy2clipboard-button

Reply:
Roger that; I'll make that change!